tests/callAsync.js

```diff
@@ -0,0 +1,3 @@
+"use strict"
+
+module.exports = typeof setImmediate === "function" ? setImmediate : setTimeout
```

Comment:
Could you move this into `tests/test-ospec.js` directly, to avoid the extra file? In this case, it could just as easily be transcluded into the file itself.

Reply:
Other than this, LGTM.